packages/code-du-travail-frontend/src/common/BigLink.js

```diff
@@ -48,9 +49,14 @@ BigLink.propTypes = {
   })
 };
 
+const { breakpoints, colors } = theme;
+
 const Icon = styled.svg`
-  width: 2em;
-  margin-right: 1em;
+  flex: 0 0 8%;
```

Comment:
These ones where not responsive so I added it here. It should have been in another PR, my bad.